app/app.component.ts

```diff
@@ -12,7 +12,11 @@ import {About} from './sections/about/about.component';
   selector: 'gapminder-app',
   encapsulation: ViewEncapsulation.None,
   directives: [Header, Footer, ...ROUTER_DIRECTIVES],
-  styles: [ <string> require('./main.styl')],
+  styles: [
+    <string> require('./main.styl'),
+    <string> require('./fonts.styl'),
+    <string> require('./variables.styl')
```

Comment:
imo the style files shouldn't be in root directory of the app, could we move it to keep or app structure more neat ?